Packs/CTF02/README.md

```diff
@@ -0,0 +1,21 @@
+Welcome to XSOAR’s Capture the Flag pack.
+
+###
+
+### **Note**
+
+Please Install `CTF 01` pack before you continue with this one.
```

Comment:
You can add the main pack as mandatory dependency

Reply:
done